### PR TITLE
Certora Formal Verification for aave L2 bridges

### PR DESCRIPTION
**Pull request by MichaelMorami**

### CODE REVIEW

File: certora/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# certora
```
Comment (miguelmtzinf) [inline certora/.gitignore:1]: Is this file needed?

**Review by miguelmtzinf — COMMENTED** [review]
- Whats the point of the `munged` directory? Is it not possible to use the contracts dependencies folder?

File: certora/.gitignore (hunk 1 above)
Comment (MichaelMorami) [reply certora/.gitignore:1]: Not really. We add these lines to the main gitignore. This is there for reference only. We can remove it if you prefer

Comment (MichaelMorami) [reply certora/.gitignore:1]: > * Whats the point of the `munged` directory? Is it not possible to use the contracts dependencies folder?

In some places in the contract we make changes like changing modifiers from internal to public, adding virtual, etc.
We use the munged folder in order to avoid polluting any of the original files, and keep all the changes contained within our own directory.
In general we try not to make too many changes by changing the original contract, in particular not any significant changes that may change the behavior of the methods.
